### PR TITLE
Add model `System::LuaScript`

### PR DESCRIPTION
**Pull request by workgena**
This PR do the following:
- create table `sys.lua_scripts`
- and corresponding model `System::LuaScripts`
- ActiveAdmin page with CRUD operations `System - Lua Scripts`
- simple validations tests `spec/models/lua_script_spec.rb`

screenshot-2018-2-9 system lua scripts
---
screenshot-2018-2-10 editing columns new name 1
---
screenshot-2018-2-9 new system lua script

### CODE REVIEW

**Comment by workgena** [comment]
I'm interesting in this:
* Do we need "Copy" action in ActiveAdmin?
* Do we need versioning on this model?
* Do we need filter by name in ActiveAdmin?
* Do we need https://highlightjs.org ?

**Comment by dmitry-sinina** [comment]
> Do we need "Copy" action in ActiveAdmin?

yes
> Do we need versioning on this model?

yes
>Do we need filter by name in ActiveAdmin?

yes
>Do we need https://highlightjs.org ?

yes

File: db/migrate/20180209140554_create_lua_scripts.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class CreateLuaScripts < ActiveRecord::Migration
+  def up
+    execute %q{
+      CREATE TABLE sys.lua_scripts (
+        id serial PRIMARY KEY,
+        name character varying NOT NULL,
```
Comment (dmitry-sinina) [inline db/migrate/20180209140554_create_lua_scripts.rb]: name should be unique

**Comment by workgena** [comment]
Done all requested changes. See updated screenshots above.
@dmitry-sinina See changes in separate commit. It this issue is finished, I will squash them into single one.

**Comment by dmitry-sinina** [comment]
Seems highlighting enabled only for Show page. Is it possible to enable it for Form input?

**Pull request merged**